lib/generators/fuel/views_generator.rb

```diff
@@ -9,7 +9,7 @@ class ViewsGenerator < Rails::Generators::Base
       source_root File.expand_path('../../../../app/views', __FILE__)
 
       def copy_views
-        directory 'fuel', 'app/views/fuel/posts'
+        directory 'fuel', 'app/views/fuel'
```

Comment:
@francirp I would keep them in the generator. I think it's valuable to customize those views. For example if someone wanted to edit the post or author form.